test/performance/odp_stash_perf.c

```diff
@@ -0,0 +1,483 @@
+/* SPDX-License-Identifier:     BSD-3-Clause
+ * Copyright (c) 2018, Linaro Limited
+ * Copyright (c) 2021, Nokia
+ * Copyright (c) 2023, Arm
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+#include <inttypes.h>
+#include <stdlib.h>
+#include <getopt.h>
+
+#include <odp_api.h>
+#include <odp/helper/odph_api.h>
+
+#define MAX_STASHS (32)
+
+typedef struct test_options_t {
+	uint32_t num_stash;
+	uint32_t num_round;
+	uint32_t max_burst;
+	uint32_t stash_size;
+	int strict;
+	int num_cpu;
+
+} test_options_t;
+
+typedef struct test_stat_t {
+	uint64_t rounds;
+	uint64_t ops;
+	uint64_t nsec;
+	uint64_t cycles;
+	uint64_t num_retry;
+
+} test_stat_t;
+
+typedef struct test_global_t {
+	odp_barrier_t    barrier;
+	test_options_t   options;
+	odp_instance_t   instance;
+	odp_shm_t        shm;
+	odp_pool_t       pool;
+	odp_stash_t	 stash[MAX_STASHS];
+	odph_thread_t    thread_tbl[ODP_THREAD_COUNT_MAX];
+	test_stat_t      stat[ODP_THREAD_COUNT_MAX];
+
+} test_global_t;
+
+static void print_usage(void)
+{
+	printf("\n"
+	       "Stash performance test\n"
+	       "\n"
+	       "Usage: odp_stash_perf [options]\n"
+	       "\n"
+	       "  -c, --num_cpu <num>    Number of worker threads. Default: 1\n"
+	       "  -n, --num_stash <num>  Number of stash. Default: 1\n"
```

Comment:
`Number of stash` -> `Number of stashes`

Reply:
Updated.